packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.15.3"
```

```suggestion
- version: "2.16.0"
```
(bump minor for enhancements)